src/main/kotlin/no/nav/syfo/behandler/BehandlerService.kt

```diff
@@ -51,6 +51,22 @@ class BehandlerService(
         return behandlere.removeDuplicates()
     }
 
+    fun searchBehandlere(
+        searchStrings: String,
+    ): List<Behandler> {
+        val tokens = searchStrings.split(" ")
+            .filter { s -> !s.isBlank() }
+            .map { s -> s.replace(",", "") }
+            .map { s -> s.replace(".", "") }
+            .map { s -> s.trim() }
+            .filter { s -> s.length > 2 }
```

Comment:
Tror det er fornuftig å *ikke* bruke søkestrenger som er mindre enn 3 tegn lange

Reply:
Per nå i frontend har vi en sjekk på `length > 3`, altså at det trengs 4 tegn før vi i det hele tatt gjør kallet til backend. Vi får bli enige om hvilken størrelse vi går for 👌🏼